src/drive/mobile/lib/reporter.js

```diff
@@ -2,16 +2,20 @@
 import Raven from 'raven-js'
 
 let isEnable = false
```

Comment:
Can we rename`isEnable` to `isEnabled` while we're at it please ? :D

Reply:
👍